.gitignore

```diff
@@ -418,3 +418,4 @@ ASALocalRun/
 .mfractor/
 
 
+385/.vscode/settings.json
```

Comment:
Should actually use this gitignore instead.

https://github.com/github/gitignore/blob/master/Global/VisualStudioCode.gitignore